ChangeLog

```diff
@@ -30,6 +30,9 @@ ver. 0.9.2 (2014/XX/XXX) - wanna-be-released
      (opposite to simply set failregex/ignoreregex that overwrites it),
      see gh-867.
    - Monit config for fail2ban in /files/monit
+   - New actions:
+     - action.d/firewallcmd-multiport Thanks Donald Yandt
```

Comment:
I guess one of them must be allports...  could be listed in a single entry,  with a single thanks  :-)